monai/data/image_reader.py

```diff
@@ -864,12 +874,19 @@ class NibabelReader(ImageReader):
     Load NIfTI format images based on Nibabel library.
 
     Args:
-        as_closest_canonical: if True, load the image as closest to canonical axis format.
-        squeeze_non_spatial_dims: if True, non-spatial singletons will be squeezed, e.g. (256,256,1,3) -> (256,256,3)
         channel_dim: the channel dimension of the input image, default is None.
             this is used to set original_channel_dim in the metadata, EnsureChannelFirstD reads this field.
             if None, `original_channel_dim` will be either `no_channel` or `-1`.
             most Nifti files are usually "channel last", no need to specify this argument for them.
+        as_closest_canonical: if True, load the image as closest to canonical axis format.
+        squeeze_non_spatial_dims: if True, non-spatial singletons will be squeezed, e.g. (256,256,1,3) -> (256,256,3)
+        to_gpu: If True, load the image into GPU memory using CuPy and Kvikio. This can accelerate data loading.
+            Default is False. CuPy and Kvikio are required for this option.
+            Note: For compressed NIfTI files, some operations may still be performed on CPU memory,
+            and the acceleration may not be significant. In some cases, it may be slower than loading on CPU.
+            #TODO: the first kvikio call is slow since it will initialize internal buffers, cuFile, GDS, etc.
```

Comment:
Hi @Nic-Ma @KumoLiu @ericspod , here I added a TODO, and later after finishing an acceleration tutorial, I will submit a separate PR to update the docstrings. What do you think?

Reply:
Is there any way to do that warm up call in some other way, such as in the constructor?

Reply:
Hi @ericspod , thanks for the suggestions.
Yes, we can do in the constructor, and trigger if `to_gpu=True`. However, there is no standard warm up way according to kvikio team's information. My current method is:
```
def warm_up():
    a = cp.arange(100)
    with tempfile.NamedTemporaryFile(delete=False) as tmp_file:
        tmp_file_name = tmp_file.name
        f = kvikio.CuFile(tmp_file_name, "w")
        # Write whole array to file
        f.write(a)
        f.close()

        b = cp.empty_like(a)
        f = kvikio.CuFile(tmp_file_name, "r")
        f.read(b)
```
I think it can be used in practical, or in a tutorial, but I'm not sure if it's formal to add this kind of code in our reader. What do you think?

Reply:
This is the probably only way if there's nothing to explicitly init things, so it's up to you if avoiding the slow initial call is worth adding this. I would add a comment about why this is here and set `delete_on_close` to `False` instead of `delete` in the context manager.

Reply:
Thanks @ericspod . I added the warm up. As for the delete/delete_on_close arg, after rerun tests, I found that we can delete in default and it will not impact the warm up performance. Therefore, now I use `tempfile.NamedTemporaryFile()`